Makefile

```diff
@@ -32,6 +32,7 @@ $(release_drivers): sht-common/sht_git_version.c
 	cp -r embedded-common/* "$${pkgdir}" && \
 	cp -r sht-common/* "$${pkgdir}" && \
 	cp -r $${driver}/* "$${pkgdir}" && \
+	cp CHANGELOG.md LICENSE "$${pkgdir}" && \
 	echo 'sensirion_common_dir = .' >> $${pkgdir}/user_config.inc && \
 	echo 'sht_common_dir = .' >> $${pkgdir}/user_config.inc && \
 	echo 'sht3x_dir = .' >> $${pkgdir}/user_config.inc && \
```

Comment:
not your fault, but this needs fixing: `echo "$${driver}_dir = ." >> `...

Reply:
I added an additional commit to fix this